### PR TITLE
Use unique term id to sort

### PR DESCRIPTION
**Pull request by makaimann**
This is most relevant for the subsumption check in IC3 variants.

### CODE REVIEW

**Comment by makaimann** [comment]
Marked `do not merge` until https://github.com/makaimann/smt-switch/pull/149 is merged, and this is pointed back to the `master` branch of `smt-switch`.

File: contrib/setup-smt-switch.sh
Comment (ahmed-irfan) [inline contrib/setup-smt-switch.sh:7]: this is commented. Is it intentional?

Comment (makaimann) [reply contrib/setup-smt-switch.sh:7]: Yeah it wasn't really necessary, but it's just until the smt-switch branch is merged.

**Review by ahmed-irfan — APPROVED** [review]

**Pull request merged**
